tabetai.coffee

```diff
@@ -16,7 +16,7 @@ open : (tabetai, target, creater) ->
     return "usage: tabetai open [target]" unless target
     if tabetai.list[target]
       tabetai.active = target
-      return "#{target} already exists. reactivate it."
+      return "#{target} already exists. Did you mean `#{robot.brain} tabetai join #{target}`?"
```

Comment:
"robot" is visible in the definition of module.exports. This code raises errors.